ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -589,7 +589,7 @@ bool UncoupledProcessesTimeLoop::loop()
         for (auto& spd : _per_process_data)
         {
             auto& pcs = spd->process;
-            auto const& x0 = *_process_solutions[pcs_idx];
+            auto& x0 = *_process_solutions[pcs_idx];
```

Comment:
sorry, I don't see why this is not const anymore. At least `preTimestep()` and `doOutput()` seem to not have changed in this PR.